lib/krane/deprecated_deploy_task.rb

```diff
@@ -250,6 +250,7 @@ def deploy_has_priority_resources?(resources)
 
     def check_initial_status(resources)
       cache = ResourceCache.new(@task_config)
+      cache.prewarm(resources)
```

Comment:
We'll need to add this to the GlobalDeployTask

Reply:
Oh yeah good shout, my bad.